package.json

```diff
@@ -33,6 +33,10 @@
     "viem": "^2",
     "wagmi": "^2"
   },
+  "dependencies": {
```

Comment:
@kyhyco this should probably be `devDependencies`.

Reply:
yes agreed

Reply:
Actually should be a `peer`